src/tests/test_magic.py

```diff
@@ -2635,3 +2637,17 @@ def test_table_does_not_exist_with_snippet_error(
 def test_negative_operations_query(ip, query, expected):
     result = ip.run_cell(query).result
     assert list(result.dict().values())[-1] == expected
+
+
+@pytest.mark.parametrize(
+    "query, expected",
+    [
+        ("%sql select json('[{\"a\":1}]')", "('[{\"a\":1}]',)"),
```

Comment:
Getting error for this case:
<img width="1055" alt="Screenshot 2024-01-17 at 7 21 52 PM" src="https://github.com/ploomber/jupysql/assets/29096135/0c84b6ca-ca28-434e-b691-e5190b34b05f">

Reply:
I think that's because you're executing the query and its expected return value together. Running the query itself,
`%sql select json('[{"a":1}]')` seems to work for me. (Removed the escape characters \ because that will return a malformed json error)